task-sdk/tests/task_sdk/definitions/test_asset_decorators.py

```diff
@@ -115,7 +115,7 @@ def root_func():
             def asset_func():
                 pass
 
-        with pytest.raises(ValueError) as err:
+        with pytest.raises(ValueError, match="nested function not supported") as err:
             root_func()
 
         assert err.value.args[0] == "nested function not supported"
```

Comment:
May be wrong, but I think err.value.args[0] is the error message.  If that is the case this (and elsewhere) can be cleaned up as well.